org.gnome.Hamster.yaml

```
@@ -0,0 +1,49 @@
+---
+app-id: org.gnome.Hamster
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: hamster
+rename-appdata-file: org.gnome.Hamster.GUI.metainfo.xml
+rename-desktop-file: org.gnome.Hamster.GUI.desktop
```

```suggestion
rename-desktop-file: org.gnome.Hamster.GUI.desktop
rename-icon: org.gnome.Hamster.GUI
```

That's why it can't find the icon